.circleci/config.yml

```diff
@@ -80,6 +80,24 @@ jobs:
           paths:
             - release/debian_bullseye
 
+  centos_7_6:
+    machine:
+      image: ubuntu-2004:202201-02
+    steps:
+      - attach_workspace:
+          at: /tmp/workspace
+      - run: docker build -f /tmp/workspace/.circleci/dockerfiles/centos-7.6 -t wake-centos-7.6 .
+      - run: |
+          cp /tmp/workspace/wake_*.tar.xz . && \
+          cp -a /tmp/workspace/tests . && \
+          docker run --rm --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined --mount type=bind,source=$PWD,target=/build wake-centos-7.6 rpmbuild -ta --define "_rpmdir /build" wake_*.tar.xz && \
```

Comment:
```
docker run --rm --device /dev/fuse --cap-add SYS_ADMIN --security-opt apparmor:unconfined --mount type=bind,source=$PWD,target=/build wake-centos-7.6 $some_command
```
I was thinking that the `source` would need to be part of $some_command
When `docker build` is running I think it runs each command in a fresh shell and doesn't preserve the state, so when we do `docker run` the sourced variables are missing